frame/instrumental-strategy-pablo/src/lib.rs

```diff
@@ -117,6 +151,13 @@ pub mod pallet {
 	pub type AssociatedVaults<T: Config> =
 		StorageValue<_, BoundedBTreeSet<T::VaultId, T::MaxAssociatedVaults>, ValueQuery>;
 
+	// TODO(belousm): where pools will be added?
+	// An asset whitelisted by Instrumental
+	// The corresponding Pool to invest the whitelisted asset into
```

Comment:
If it is docs, than the line should start with `///` (three times slash), if not, than you can free to resolve the comment.